connector/__init__.py

```diff
@@ -0,0 +1,8 @@
+from connector.connector import Connector
+from connector.db_structures import NotebookDb, CellDb, CodeCellDb, MdCellDb
+from connector.db_structures import create_db
```

Comment:
from connector.connector import Connector
from connector.db_structures import NotebookDb, CellDb, CodeCellDb, MdCellDb
from connector.db_structures import create_db

```suggestion
from .connector import Connector
from .db_structures import NotebookDb, CellDb, CodeCellDb, MdCellDb
from .db_structures import create_db
```